halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/node/PersistentStorage.java

```diff
@@ -1,22 +1,25 @@
 package com.netflix.spinnaker.halyard.config.model.v1.node;
 
-import com.netflix.spinnaker.halyard.config.model.v1.providers.google.GoogleProvider;
+import com.netflix.spinnaker.halyard.config.model.v1.persistentStorage.CassandraPersistentStore;
+import com.netflix.spinnaker.halyard.config.model.v1.persistentStorage.GcsPersistentStore;
+import com.netflix.spinnaker.halyard.config.model.v1.persistentStorage.S3PersistentStore;
 import com.netflix.spinnaker.halyard.config.problem.v1.ConfigProblemSetBuilder;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
-import lombok.extern.slf4j.Slf4j;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.stream.Collectors;
+import java.lang.reflect.Field;
+import java.util.Arrays;
+import java.util.Optional;
 
-/**
- * This is the configuration for S3/GCS storage options.
- */
 @Data
 @EqualsAndHashCode(callSuper = false)
-@Slf4j
 public class PersistentStorage extends Node {
+  CassandraPersistentStore cassandra = new CassandraPersistentStore() {{
```

Comment:
I only included it so that we could have a default value. Otherwise your config will start out in a 'broken' state and you'll have to fix it before validation passes. Is there some better default or are you fine with that?